### PR TITLE
Tests for country

### PR DESCRIPTION
**Pull request by vicmaburrito**
- [x] Use React documentation.
- [x] Use React components.
- [x] Use React props.
- [x] Use React Router.
- [x] Connect React and Redux.
- [x] Handle events in a React app.
- [x] Write integration tests with a React testing library.
- [x] Use styles in a React app.
- [x] Use React life cycle methods.
- [x] Apply React best practices and language style guides in code.
- [x] Use store, actions, and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,56 @@
...
+<img width="310" alt="rockets preview" src="/Readme-assets/1643923228204.png"><br>
+<img width="310" alt="rockets preview" src="/Readme-assets/1643924176103.png"><br>
+
+<h2>Live Demo</h2>
+
+[Covid Tracker](https://youthful-turing-b029e9.netlify.app/)
+
+<h2>Presentation</h2>
+
+[Loom video](https://youthful-turing-b029e9.netlify.app/)
```
Comment (bimbolabuari) [inline README.md]: Kindly add the correct video presentation url as required 🙂

File: src/Components/Countries/Country.js [hunk 2]
```diff
@@ -22,7 +22,8 @@ const Country = ({ country }) => (
 );
 
 Country.propTypes = {
-  country: PropTypes.instanceOf(Object).isRequired,
+  // eslint-disable-next-line react/require-default-props
```
Comment (bimbolabuari) [inline src/Components/Countries/Country.js]: Let's remove all disabled linters, Linting makes the code cleaner and professional 👍 

Kindly check for other occurrences 🙂

File: README.md [hunk 3]
```diff
@@ -29,42 +70,30 @@ Your app is ready to be deployed!
...
+Feel free to check the <a href="https://github.com/vicmaburrito/COVID-19-TRACKING/issues" rel="noopener noreferrer">issues page</a>.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+<h2>Show your support</h2>
 
-### Advanced Configuration
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+<!-- <h2>Acknowledgments</h2>
```
Comment (bimbolabuari) [inline README.md]: Let's give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the acknowledgement section ✊

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @vicmaburrito , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Deploys the application using Netlify
- The project contains a Home page and Detail page.
- The project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Review by rukundoeric — APPROVED** [review]
Hi @vicmaburrito,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**